src/stories/Library/link-with-icon/LinkWithIcon.tsx

```diff
@@ -0,0 +1,51 @@
+import clsx from "clsx";
+import { ReactComponent as ArrowSmallRight } from "../Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg";
+import { IconTypes, LinkTypes } from "./LinkTypes";
+
+export type LinksProps = {
+  href: string;
+  linkText: string;
+  linkType: LinkTypes;
+};
+
+const linkTypeAttributes: {
+  [key in LinkTypes]: {
+    target: "_self" | "_blank";
+    icon: IconTypes;
+    folder: "basic" | "collection";
+  };
+} = {
+  internal: { target: "_self", icon: "ArrowBack", folder: "collection" },
+  external: {
+    target: "_blank",
+    icon: "icon-external-link",
+    folder: "basic",
+  },
+  download: { target: "_blank", icon: "Ebook", folder: "collection" },
+  search: { target: "_self", icon: "SearchBooks", folder: "collection" },
+};
+
+export const LinkWithIcon = ({ href, linkText, linkType }: LinksProps) => {
+  const { target, icon, folder } = linkTypeAttributes[linkType];
+
+  const linkIconClass = clsx({
+    "rotate-180": linkType === "internal",
+  });
+
+  return (
+    <a
+      href={href}
+      target={target}
+      rel={target === "_blank" ? "noopener noreferrer" : undefined}
```

Comment:
Please consider if there is a reason for us to set `noopener` and/or `noreferrer`.

[According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel) `noreferrer` has the same effect as `noopener` so if we set `noreferrer`, `noopener` can be removed.

Also: Do we want to set either?

Reply:
I removed it entirely then. 

Noopener was indeed redundant, and i guess if we set and external link we are fine with sending the referer information along.